src/main/cpp/subsystems/StateSubsystem.cpp

```diff
@@ -72,15 +72,24 @@ frc2::CommandPtr StateSubsystem::RunState() {
 frc2::CommandPtr StateSubsystem::StartIntaking() {
   return m_subsystems.led->ShowFromState([] { return RobotState::Intaking; })
       .AndThen(
-          IntakeIn(m_subsystems.intake)
-              .ToPtr()
+          IntakingCommands::Intake(m_subsystems.intake)
               .RaceWith(DriveVelocity(0_deg, 2_mps, m_subsystems.drive)
                             .ToPtr()
                             .Repeatedly())
+              .Until([this] {
+                return m_subsystems.intake->NotePresentUpper() &&
+                       m_subsystems.intake->NotePresentLower();
+              })
+              .FinallyDo([this] {
```

Comment:
Still need to figure out why this doesn't stop. Is it because of the `Repeatedly` on `DriveVelocity`? Maybe have it drive continuously on its own and then stop when acted upon externally?